common/tools/eslint-plugin-azure-sdk/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
do we need another LICENSE file? I think the one at the root of the mult-repo is fine

Reply:
They're all over core and storage, but we don't need to have one I wouldn't think. I can just remove this one and then we can talk about removing the others.

```
$ find sdk -mindepth 4 -maxdepth 4 -name "LICENSE"
sdk/cosmosdb/cosmos/LICENSE
sdk/core/abort-controller/LICENSE
sdk/core/core-paging/LICENSE
sdk/core/core-http/LICENSE
sdk/core/core-asynciterator-polyfill/LICENSE
sdk/core/logger/LICENSE
sdk/core/core-amqp/LICENSE
sdk/test-utils/recorder/LICENSE
sdk/storage/storage-file-share/LICENSE
sdk/storage/storage-queue/LICENSE
sdk/storage/storage-blob/LICENSE
sdk/storage/storage-file-datalake/LICENSE
common/tools/eslint-plugin-azure-sdk/LICENSE
```